platform/broadcom/docker-syncd-brcm/start.sh

```diff
@@ -1,8 +1,15 @@
 #!/usr/bin/env bash
 
+PLATFORM_DIR=/usr/share/sonic/platform
+
 rm -f /var/run/rsyslogd.pid
 
 supervisorctl start rsyslogd
 
 supervisorctl start syncd
 
+# If this platform has an initialization file for the Broadcom LED microprocessor, load it
+if [ -r ${PLATFORM_DIR}/led_proc_init.soc ]; then
+    /usr/bin/bcmcmd "rcload ${PLATFORM_DIR}/led_proc_init.soc"
```

Comment:
when syncd is started, the bcmshell may not be available immediately. maybe it is better to modify the sai code to load the soc after the init is finished.